### PR TITLE
[paho-mqttpp3] update to 1.1

### PR DESCRIPTION
**Pull request by hank121314**
Update paho.mqtt.cpp to version 1.1.

I cannot use ssl feature if target_link_libraries contains non ssl library.
So I use PAHO_WITH_SSL to specific which library should be targeted.

### CODE REVIEW

File: ports/paho-mqttpp3/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: paho-mqttpp3
-Version: 1.0.1-3
+Version: 1.1
 Description: Paho project provides open-source C++ wrapper for Paho C library
```
Comment (LilyWangL) [inline ports/paho-mqttpp3/CONTROL:3]: Can you please add the Homepage for this port in the CONTROL file?

**Review by LilyWangL — CHANGES_REQUESTED** [review]

File: ports/paho-mqttpp3/portfile.cmake [hunk 2]
```diff
@@ -4,8 +4,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
[the remark this answers is not in the document]
Comment (hank121314) [reply ports/paho-mqttpp3/portfile.cmake:4]: OK. Thanks for your reminder!

**Comment by strega-nil** [comment]
Alright, this looks good to me! Thanks @hank121314 :)

**Pull request merged**